presto-docs/src/main/sphinx/connector/hive-security.rst

```diff
@@ -20,6 +20,10 @@ Property Value                                     Description
                                                    metadata, such as ``CREATE``, ``INSERT`` or ``DELETE``, are
                                                    allowed.
 
+``file``                                           Authorization checks are enforced by the file specified by
+                                                   ``security.config-file``. See below for the format of this
```

Comment:
... are enforced using a config file specified by the Hive configuration property ...